src/pages/home/ui/HomePageSection/index.tsx

```
@@ -187,21 +160,20 @@ export function SettlementList() {
         <S.SettlementListWrapper>
-          {settlementList.map((data) => (
+          {settlementList.map((item) => (
             <HomeExpenseItem
-              key={data.id}
-              date={data.date}
-              groupName={data.groupName}
-              totalAmount={data.totalAmount}
-              paidMember={data.paidMember}
-              totalMember={data.totalMember}
+              key={item.groupId}
+              date={format(new Date(item.createdAt), 'yyyy년 M월 d일', { locale: ko })}
```

ESLint

[error] Replace `·locale:·ko` with `⏎················locale:·ko,⏎·············`

(prettier/prettier)